docs/faq.mdx

```diff
@@ -10,7 +10,7 @@ This is a list of frequently asked questions for SideStore. This has a list of c
 
 ## What is SideStore?
 
-SideStore is an iOS application that allows you to sideload apps onto your iOS device using only your Apple ID. SideStore resigns apps with your personal development certificate, and then uses a specially designed VPN in order to trick iOS into installing them. SideStore will periodically "refresh" your apps in the background, to keep their normal 7-day development period from expiring. SideStore's goal is to provide an untethered sideloading experience. It's a community driven fork of AltStore, and has already implemented some of the community's most-requested features. You only need a computer once during installation.
```

Comment:
But that's just what we're doing, we are tricking iOS into thinking there is a Mac doing the sideloading.

I think this change should stay

cc: @SternXD

Reply:
👍